### PR TITLE
Fix copyright issues when using diameterc

### PR DESCRIPTION
**Pull request by anatoliegolovco**
With old copyright, even if compiling self generated .dia files, all
rights belong to Ericsson AB

### CODE REVIEW

File: lib/diameter/src/compiler/diameter_codegen.erl [hunk 1]
```diff
@@ -792,17 +792,23 @@ header() ->
...
+     "%% %CopyrightBegin%\n"
      "%%\n"
-     "%% The information in this document is the property of Ericsson.\n"
+     "%% Copyright Ericsson AB 2010-2015. All Rights Reserved.\n"
      "%%\n"
-     "%% Except as specifically authorized in writing by Ericsson, the\n"
-     "%% receiver of this document shall keep the information contained\n"
-     "%% herein confidential and shall protect the same in whole or in\n"
-     "%% part from disclosure and dissemination to third parties.\n"
+     "%% Licensed under the Apache License, Version 2.0 (the "License");\n"
```
Comment (fenollp) [inline lib/diameter/src/compiler/diameter_codegen.erl]: Unescaped double quotes

File: lib/diameter/src/compiler/diameter_codegen.erl [hunk 2]
```diff
@@ -792,17 +792,23 @@ header() ->
...
+     "%% you may not use this file except in compliance with the License.\n"
+     "%% You may obtain a copy of the License at\n"
      "%%\n"
-     "%% Disclosure and disseminations to the receivers employees shall\n"
-     "%% only be made on a strict need to know basis.\n"
+     "%%     http://www.apache.org/licenses/LICENSE-2.0\n"
+     "%%\n"
+     "%% Unless required by applicable law or agreed to in writing, software\n"
+     "%% distributed under the License is distributed on an "AS IS" BASIS,\n"
```
Comment (fenollp) [inline lib/diameter/src/compiler/diameter_codegen.erl]: Same here

**Comment by otp-anders** [comment]
Thanks. This header predates diameter's inclusion in OTP, and should have
been removed or changed long ago Is there any need for the header at all in
generated code?

Anders



On Thu, Aug 11, 2016 at 10:47 AM, Anatolie Golovco <notifications@github.com

**Comment by OTP-Maintainer** [comment]
Patch has passed first testings and has been assigned to be reviewed


----------------

I am a script, I am not human 

----------------

**Pull request closed without merging**